### ISSUE TITLE
Confusing schemas for `themes`

### ISSUE THREAD

**Issue by m-mohr** [opengeospatial/ogcapi-records#235]
I just searched for information about `themes` and found various schemas and examples, which was a bit confusing:

Maybe check the following files whether they are up-to-date:
- themes is an array of URIs? Looks wrong: https://github.com/opengeospatial/ogcapi-records/blob/master/core/standard/clause_12_local_resources_catalogue.adoc#example
- type: datetime? What's that? https://github.com/opengeospatial/ogcapi-records/blob/master/core/examples/json/queryables.json

These look consistent:
- https://github.com/opengeospatial/ogcapi-records/blob/master/core/examples/json/record.json
- https://github.com/opengeospatial/ogcapi-records/blob/master/core/examples/json/keywords-themes.json
- https://github.com/opengeospatial/ogcapi-records/blob/master/core/openapi/schemas/catalogue.yaml
- https://github.com/opengeospatial/ogcapi-records/blob/master/core/openapi/schemas/recordGeoJSON.yaml
- https://github.com/opengeospatial/ogcapi-records/blob/master/core/openapi/ogcapi-records-1.yaml

**Comment by pvretano** [comment]
@m-mohr yeah, the examples are not quite up to date relative to the schemas.  I have it on my todo list to update and augment them.

**Comment by m-mohr** [comment]
@pvretano Thanks. The "good" schemas itself also have an issue, I think. Neither `concepts` nor `scheme` is required. I assume both should be required?

In the STAC extension I made the schema a bit more strict in general:
```json
{
          "type": "array",
          "minItems": 1,
          "items": {
            "type": "object",
            "required": [
              "concepts",
              "scheme"
            ],
            "properties": {
              "concepts": {
                "type": "array",
                "items": {
                  "type": "object",
                  "required": [
                    "id"
                  ],
                  "properties": {
                    "id": {
                      "type": "string",
                      "minLength": 1
                    },
                    "title": {
                      "type": "string",
                      "minLength": 1
                    },
                    "description": {
                      "type": "string",
                      "minLength": 1
                    },
                    "url": {
                      "type": "string",
                      "format": "uri"
                    }
                  }
                }
              },
              "scheme": {
                "type": "string",
                "minLength": 1
              }
            }
          }
        }
```

**Comment by pvretano** [comment]
@m-mohr I'll have to check with the author of that schema about why `concepts` and `scheme` are not required.  Not exactly sure why that is the case ... makes sense that they should be otherwise there is little distinction with keywords.  Thanks for the heads up.

**Comment by tomkralidis** [comment]
Thanks @m-mohr.  PR in #237

**Comment by m-mohr** [comment]
Great, the requirements are fixed. Now it's just to clean-up some older files, right?

> * themes is an array of URIs? Looks wrong: https://github.com/opengeospatial/ogcapi-records/blob/master/core/standard/clause_12_local_resources_catalogue.adoc#example
> 
> * type: datetime? What's that? https://github.com/opengeospatial/ogcapi-records/blob/master/core/examples/json/queryables.json

**Comment by tomkralidis** [comment]
Thanks @m-mohr.  Associated PR in #236.

**Comment by m-mohr** [comment]
@tomkralidis Thanks, but I don't see any changes in #236 to the two files mentioned in https://github.com/opengeospatial/ogcapi-records/issues/235#issuecomment-1528834725 ?!

**Comment by m-mohr** [comment]
Oh, I guess that's a typo and meant #246 :-)

**Comment by tomkralidis** [comment]
Oops mea culpa, yes #246 :)

**Comment by tomkralidis** [comment]
Given #246 is now merged, I think we can close this one @pvretano @m-mohr ?

**Comment by pvretano** [comment]
Resolved by PR #246 ... closing.  Create a new issue or re-open this one if something comes up.
